xgboost_ray/main.py

```diff
@@ -411,13 +413,15 @@ def _create_actor(rank: int,
                   num_cpus_per_actor: int,
                   num_gpus_per_actor: int,
                   resources_per_actor: Optional[Dict] = None,
+                  placement_group: Optional[PlacementGroup] = None,
                   queue: Optional[Queue] = None,
                   checkpoint_frequency: int = 5) -> ActorHandle:
 
     return RayXGBoostActor.options(
         num_cpus=num_cpus_per_actor,
         num_gpus=num_gpus_per_actor,
-        resources=resources_per_actor).remote(
+        resources=resources_per_actor,
+        placement_group=placement_group).remote(
```

Comment:
@rkooo567 do we still have to pass in `num_cpus`, `num_gpus`, `resources` here if we are also passing in the placement group?

Reply:
I think so, yeah